dev-packages/e2e-tests/test-applications/create-remix-app-v2/package.json

```diff
@@ -12,18 +12,18 @@
   },
   "dependencies": {
     "@sentry/remix": "latest || *",
-    "@remix-run/css-bundle": "2.0.0",
-    "@remix-run/node": "2.0.0",
-    "@remix-run/react": "2.0.0",
-    "@remix-run/serve": "2.0.0",
+    "@remix-run/css-bundle": "2.7.2",
+    "@remix-run/node": "2.7.2",
+    "@remix-run/react": "2.7.2",
+    "@remix-run/serve": "2.7.2",
     "isbot": "^3.6.8",
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
     "@playwright/test": "^1.36.2",
-    "@remix-run/dev": "2.0.0",
-    "@remix-run/eslint-config": "2.0.0",
+    "@remix-run/dev": "2.7.2",
+    "@remix-run/eslint-config": "2.7.2",
```

Comment:
Also updated this to latest, to make sure the `ErrorBoundary` works with the latest Remix in `production` mode.